src/main/kotlin/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.kt

```diff
@@ -0,0 +1,675 @@
+            return informatieobjectConverter.convertUUIDsToREST(it, zaak)
+        } ?: run {
+            checkNotNull(zaak) { "Zoekparameters hebben geen waarde" }
+            assertPolicy(policyService.readZaakRechten(zaak).lezen)
+            var enkelvoudigInformatieobjectenVoorZaak = listEnkelvoudigInformatieobjectenVoorZaak(zaak)
+            if (zoekParameters.gekoppeldeZaakDocumenten) {
+                enkelvoudigInformatieobjectenVoorZaak.addAll(listGekoppeldeZaakInformatieObjectenVoorZaak(zaak))
+            }
+            zoekParameters.besluittypeUUID?.let {
+                val besluittype = ztcClientService.readBesluittype(zoekParameters.besluittypeUUID)
+                val compareList = besluittype.informatieobjecttypen.stream().map { UriUtil.uuidFromURI(it) }.toList()
+                enkelvoudigInformatieobjectenVoorZaak = enkelvoudigInformatieobjectenVoorZaak.filter {
+                    compareList.contains(it.informatieobjectTypeUUID)
+                }.toMutableList()
+            }
+            return enkelvoudigInformatieobjectenVoorZaak
+        }
+    }
+
+    @GET
+    @Path("informatieobjecten/zaak/{zaakUuid}/teVerzenden")
+    fun listEnkelvoudigInformatieobjectenVoorVerzenden(
+        @PathParam("zaakUuid") zaakUuid: UUID
+    ): List<RESTEnkelvoudigInformatieobject> {
+        val zaak = zrcClientService.readZaak(zaakUuid)
+        assertPolicy(policyService.readZaakRechten(zaak).lezen)
+        return zrcClientService.listZaakinformatieobjecten(zaak).stream()
+            .map { it.informatieobject }
+            .map(drcClientService::readEnkelvoudigInformatieobject)
+            .filter(::isVerzendenToegestaan)
+            .map { informatieobjectConverter.convertToREST(it, zaak) }
+            .toList()
+    }
+
+    @POST
+    @Path("informatieobjecten/verzenden")
+    fun verzenden(gegevens: RESTDocumentVerzendGegevens) {
+        val informatieobjecten = gegevens.informatieobjecten.stream()
+            .map(drcClientService::readEnkelvoudigInformatieobject)
+            .toList()
+        val zaak = zrcClientService.readZaak(gegevens.zaakUuid)
+        assertPolicy(policyService.readZaakRechten(zaak).wijzigen)
+        informatieobjecten.forEach {
+            assertPolicy(isVerzendenToegestaan(it))
+        }
+        informatieobjecten.forEach {
+            enkelvoudigInformatieObjectUpdateService.verzendEnkelvoudigInformatieObject(
+                URIUtil.parseUUIDFromResourceURI(it.url),
+                gegevens.verzenddatum,
+                gegevens.toelichting
+            )
+        }
+    }
+
+    @POST
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("informatieobject/{zaakUuid}/{documentReferenceId}")
+    fun createEnkelvoudigInformatieobjectAndUploadFile(
+        @PathParam("zaakUuid") zaakUuid: UUID,
+        @PathParam("documentReferenceId") documentReferenceId: String,
+        @QueryParam("taakObject") taakObject: Boolean,
```

Comment:
```suggestion
        @QueryParam("taakObject") isTaakObject: Boolean,
```
? makes it clearer what it is